.xe/specs/blueprint/spec.md

```diff
@@ -0,0 +1,413 @@
+
+- **NFR-3**: Reliability
+  - Dependency graph MUST be validated for cycles before merging
+  - Feature IDs MUST be unique across all phases
+
+- **NFR-4**: Performance
+  - Blueprint spec MUST be parseable by AI in single pass without external dependencies
+
+- **NFR-5**: Observability
+  - Rollout plan tracks feature implementation status for visibility
+  - Phase assignments enable filtering features by maturity level
+
+- **NFR-6**: Accessibility
+  - Blueprint MUST be human-readable markdown suitable for both AI and human review
+  - Mermaid graph MUST render correctly in GitHub for visual dependency understanding
+
+- **NFR-7**: Globalization
+  - Not applicable - Catalyst is English-only framework
+
+- **NFR-8**: Backward compatibility
+  - Phase-relative tier numbering prevents breaking changes when phases evolve
+  - Feature IDs SHOULD remain stable across blueprint updates
+
+## Key Entities
+
+Entities owned by this feature:
+
+- **Blueprint Specification**: Complete feature roadmap in `.xe/specs/blueprint/spec.md` with all features, dependencies, phases, and priorities
+- **Feature**: Discrete capability with ID, name, description, phase, tier, dependencies, complexity, and priority
+- **Feature Dependency Graph**: Acyclic directed graph showing which features depend on others
+- **Phase**: Strategic grouping of features (POC, Mainstream, Innovation, Platform, Enterprise)
+- **Tier**: Dependency-based grouping within a phase (e.g., 1.1, 1.2, 1.3) where features in same tier can be built in parallel
+
+Entities from other features:
+
+- **Context Files** (product-context, engineering-context): Files that features read/write to understand project requirements
+- **Playbooks** (playbook-engine): Structured workflows that features execute
+- **Rollout Plans** (feature-rollout): Orchestration plans that coordinate feature implementation
+
+Inputs:
+
+- GitHub issue #41 content with phased implementation details
+- `.xe/product.md` for product strategy and phase priorities
+- `.xe/architecture.md` for technical patterns and constraints
+- `.xe/engineering.md` for design principles (KISS, YAGNI, SRP, DRY, etc.)
+- Existing source code structure for understanding current implementation state
+
+Outputs:
+
+- `.xe/specs/blueprint/spec.md` with complete feature breakdown
+- `.xe/specs/blueprint/research.md` with analysis and decision log
+- `.xe/specs/blueprint/plan.md` with feature breakdown methodology
+- `.xe/specs/blueprint/tasks.md` with steps to populate spec
+- `.xe/rollouts/rollout-blueprint.md` with feature status tracking
+- Feature dependency graph in mermaid format
+
+## Dependencies
+
+- This is a meta-feature that defines all other features, so it has no dependencies
+- However, it MUST read existing context files (product.md, architecture.md, engineering.md) to inform feature breakdown
+- GitHub integration (gh CLI or API access) is required to fetch issue #41 content
+
+## Core Entities
```

Comment:
Why are core entities defined separately from key entities? Is this a duplication in start-blueprint or the spec.md template? One or both of those need to be updated if they are guiding AI to create this duplicate section.